transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/util/extensions/MessageExtension.kt

```diff
@@ -105,19 +105,48 @@ private fun List<StructuredMessage.Content>.toQuickReplies(): List<ButtonRespons
     }
 }
 
+private fun StructuredMessage.Content.Action.toMessageCardAction(): Message.Card.Action =
+    when (type.lowercase()) {
```

Comment:
why you do lowercase on the `type`?
Also, I have a feeling that this entire function is not fully covered with tests. Do we have test for `else` condition?